feat(docs): update backport documentation (#1989)
spinnaker/spinnaker.github.io: community/releases/release-manager-runbook/index.md

Repeat weeklyish for each supported version.

1. Audit [backport candidates](#audit-backport-candidates).

> To view what's been merged into the each release branch since the last release,

Reviewer: ```suggestion
To view what's been merged into each release branch since the last release,
```

Repeat weekly.

1. Audit each PR that has been labelled a
[backport candidate](https://github.com/pulls?q=org%3Aspinnaker+is%3Apr+sort%3Aupdated-desc+label%3Abackport-candidate).

1. If a candidate meets the
[release branch patch criteria](/community/contributing/releasing#release-branch-patch-criteria):

    1. Remove the `backport-candidate` label from the PR.
    
    1. Add a comment instructing
       [Mergify](https://doc.mergify.io/commands.html#backport) to create
       backport PRs against one or more release branches. For example, to
       create backport PRs against the 1.20 and 1.21 release branches, comment:
       
       > @Mergifyio backport release-1.20.x release-1.21.x

    1. Approve and merge the backport PRs.

    1. If Mergify cannot create a backport because there are merge conflicts,
       ask the contributor to open a PR against the target release branches with
       their commits manually
       [cherry-picked](https://git-scm.com/docs/git-cherry-pick).

1. If a candidate does not meet the
[release branch patch criteria](/community/contributing/releasing#release-branch-patch-criteria),
add an explanation to the contributor as a comment.

>     1. If it is impossible for the candidate to meet the criteria (e.g., it does

Reviewer: ```suggestion
    1. If it's impossible for the candidate to meet the criteria (for example, it doesn't
```

>        not fix a regression), remove the `backport-candidate` label.

Reviewer: ```suggestion
       fix a regression), remove the `backport-candidate` label.
```

>     1. If the contributor can amend the candidate to meet the criteria (e.g.,

Reviewer: ```suggestion
    1. If the contributor can amend the candidate to meet the criteria (for example,
```

>        add test coverage), do not remove the `backport-candidate` label.

Reviewer: ```suggestion
       add test coverage), don't remove the `backport-candidate` label.
```
